src/transformers/modeling_longformer.py

```diff
@@ -1132,8 +1169,8 @@ def forward(
         from transformers import LongformerTokenizer, LongformerForTokenClassification
         import torch
 
-        tokenizer = LongformerTokenizer.from_pretrained('longformer-base-4096')
-        model = LongformerForMultipleChoice.from_pretrained('longformer-base-4096')
+        tokenizer = LongformerTokenizer.from_pretrained('allenai/longformer-base-4096')
+        model = LongformerForMultipleChoice.from_pretrained('allenai/longformer-base-4096')
         choices = ["Hello, my dog is cute", "Hello, my cat is amazing"]
```

Comment:
maybe use an example that has `context` in addition to the `choices` with a comment showing where the automatic global attention is going to be.

Reply:
very good point!